samples/sqltransport/store-and-forward/Version_2/Sender/ForwardBehavior.cs

```diff
@@ -40,7 +40,7 @@ public void Invoke(IncomingContext context, Action next)
     public class Registration : RegisterStep
     {
         public Registration()
-            : base("OutboxLoopbackReceive", typeof(OutboxLoopbackReceiveBehavior), "OutboxLoopbackReceive")
+            : base("Forward", typeof(ForwardBehavior), "Forwards the message to the destination.")
         {
             InsertBefore(WellKnownStep.LoadHandlers);
```

Comment:
Is this targeting v6? (if yes you can remove the before since the handlers are in a different stage)